lightblue-ldap-crud/src/main/java/com/redhat/lightblue/crud/ldap/LdapCRUDController.java

```diff
@@ -533,5 +530,22 @@ private void execute(CRUDOperationContext ctx, ExecutionHandler handler){
         abstract void onSuccess(LDAPResult result);
 
     }
-
-}
+    
+    @Override
+	public LightblueHealth checkHealth() {
+		boolean isHealthy = true;
+		List<LDAPConnection> ldapConnnections = dbResolver.getConnections();
+		List<String> details = new ArrayList<>(ldapConnnections.size());
+
+		for (LDAPConnection ldapConnection : ldapConnnections) {
+			if (ldapConnection == null || !ldapConnection.isConnected()) {
+				isHealthy = false;
+				details.add(ldapConnection + ":ldap_connection_ok!=1");
```

Comment:
here if we have the full exception we could add some of that detail to the response.